server/tools/hatohol-db-initiator.in

```diff
@@ -132,6 +132,11 @@ def parse_default_conf():
 
 def get_default_conf_item(item_name):
     config = parse_default_conf()
+    if config is None:
+       global not_found_config_file
+       not_found_config_file = True
+       return "<Unknown>"
```

Comment:
How about adding default dictionaries for them?

```
{ hatohol_db_user: "hatohol", hatohol_db_password: "hatohol" }
```

I think it's better than `<Unknown>` because they are default values in the previous implementation.